Web/Edubase.Web.UI/Views/Tools/IndSchoolSearchDownload/ReadyToDownload.cshtml

```diff
@@ -53,7 +53,7 @@
                 <p>Your requested file is ready to download.</p>
 
                 <p>
-                    <input type="submit" class="govuk-button" data-module="govuk-button" value="@name" /><br />
+                    <input type="submit" id="@StringUtils.ElementIdFormat(@name)-button" class="govuk-button" data-module="govuk-button" value="@name" /><br />
```

Comment:
I don't have the knowledge to know that the variable 'name' will always contain a string that can be successfully formatted to produce a valid id here - but if you know that I'm happy.

Reply:
the @name is referred to the name of the download

Reply:
Thanks @CathLass yes the name value will always contain one of two strings which can be formatted successfully.

Reply:
For further info/context using a screenshot from @JaswiniP (thanks!) it seems that `name` refers to the file type name from a constrained list of values, which should never have unusual characters 👍 

![image](https://user-images.githubusercontent.com/96429508/213690703-19ab738c-312b-48c2-85bd-9525c83c3d49.png)